This patch hardcodes the installation prefix to /usr and updates
the location of the man pages from $PREFIX/man to $PREFIX/share/man.

--- cowsay-3.03/install.sh.orig	2020-06-05 13:39:36.615616000 +0000
+++ cowsay-3.03/install.sh	2020-06-05 13:40:24.178066800 +0000
@@ -67,7 +67,7 @@
 echo Okay, time to install this puppy.
 
 echo s,%BANGPERL%,!$usethisperl,\; > install.pl
-echo s,%PREFIX%,$PREFIX,\; >> install.pl
+echo s,%PREFIX%,/usr,\; >> install.pl
 set -x
 mkdir -p $PREFIX/bin || (mkdir $PREFIX; mkdir $PREFIX/bin)
 $usethisperl -p install.pl cowsay > $PREFIX/bin/cowsay
@@ -74,8 +74,8 @@
 chmod a+x $PREFIX/bin/cowsay
 ln -s cowsay $PREFIX/bin/cowthink
-mkdir -p $PREFIX/man/man1 || ($mkdir $PREFIX; mkdir $PREFIX/man; mkdir $PREFIX/man/man1)
-$usethisperl -p install.pl cowsay.1 > $PREFIX/man/man1/cowsay.1
-chmod a+r $PREFIX/man/man1/cowsay.1
-ln -s cowsay.1 $PREFIX/man/man1/cowthink.1
+mkdir -p $PREFIX/share/man/man1 || ($mkdir $PREFIX; mkdir $PREFIX/man; mkdir $PREFIX/man/man1)
+$usethisperl -p install.pl cowsay.1 > $PREFIX/share/man/man1/cowsay.1
+chmod a+r $PREFIX/share/man/man1/cowsay.1
+ln -s cowsay.1 $PREFIX/share/man/man1/cowthink.1
 mkdir -p $PREFIX/share/cows || (mkdir $PREFIX; mkdir $PREFIX/share; mkdir $PREFIX/share/cows)
 tar -cf - $filelist | (cd $PREFIX/share && tar -xvf -)
